### ISSUE TITLE
plot.PLNPCA_fit does not pass arguments appropriately

### ISSUE THREAD

**Issue by jchiquet** [jchiquet/PLNmodels#56]
Consider the PCA example from the vignette

```
library(PLNmodels)
data(trichoptera)
trichoptera <- prepare_data(trichoptera$Abundance, trichoptera$Covariate)
PCA_models <- PLNPCA(
  Abundance ~ 1 + offset(log(Offset)),
  data  = trichoptera,
  ranks = 1:5
)
myPCA_ICL <- getBestModel(PCA_models, "ICL")
```

This work for plotting individual map for axes (1,3)

```
myPCA_ICL$plot_individual_map(axes = c(1,3))  
```

But the following does not work !

```
plot(myPCA_ICL, map = "individual", axes = c(1,3))  
```

**Comment by mahendra-mariadassou** [comment]
I can't reproduce the problem (`$plot_individual_map()` and `plot()` produce the same graphs when I try). Which version of PLNmodels are you using? 

In the CRAN version, `$plot_individual_map()` calls `get_ggplot_ind_map()`
https://github.com/jchiquet/PLNmodels/blob/bd4156b4eb1a91a8221df327cfa255bd73110503/R/plot_utils.R#L1-L20
which produces either a 2D-scatterplot or a density plot but not multiple graphs (unlike `$plot_PCA()`)

It has not changed since (according to the blame). Should I change `$plot_individual_map()` to make multiple calls to `get_ggplot_ind_map()` when there are 3 axes or more?

**Comment by mahendra-mariadassou** [comment]
My bad, I was sloppy when reading the message. Working on it.

**Comment by mahendra-mariadassou** [comment]
Fixed in b0e678f94a415da6399245cf7070059a379be24f
